torch/csrc/jit/python_ir.cpp

```diff
@@ -451,9 +451,9 @@ void initPythonIRBindings(PyObject * module_) {
     ;
 
   py::class_<DynamicType, Type, std::shared_ptr<DynamicType>>(m, "DynamicType")
-    .def(py::init<>());
+    .def(py::init([](){ return DynamicType::create(); }));
```

Comment:
Why does `DynamicType` have a `create` method? Shouldn't you just call `get()`?

Reply:
I thought about it, the place pybind11 did here is to bind the constructor, and we are using a factor create as a custom constructor, so I think it make more sense for us to leave it with the `create` rather than `get()`. In anyways we are calling `create` inside `get`.